.github/workflows/ar-chromatic.yml

```
@@ -0,0 +1,51 @@
+name: AR Chromatic 👓
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  chromatic:
+    name: Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout - On Pull Request
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'pull_request'}}
+        with:
+          fetch-depth: 0
+          # By default the pull_request event will run on a ephermeral merge commit which simulates a merge between the pull request
+          # and the target branch. This can cause issues with Chromatic https://www.chromatic.com/docs/turbosnap#github-pullrequest-triggers
+          # Hopefully by checking out the HEAD commit of a PR instead of the merge commit we can avoid some of those issues.
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Checkout - On Push Event
+        uses: actions/checkout@v3
+        if: ${{ github.event_name == 'push'}}
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+      # Cache npm dependencies using https://github.com/bahmutov/npm-install
+      # Root yarn installs all workspaces (root, common, dotcom)
+      - uses: bahmutov/npm-install@v1
```

we need to point the nvmrc file with the official node setup, and can do away with the bumatov caching action now too:

```suggestion
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
```